sky/utils/accelerator_registry.py

```diff
@@ -1,9 +1,21 @@
-"""Resources related utilities."""
+"""Accelerator registry."""
 from sky.clouds import service_catalog
 from sky.utils import ux_utils
 
-# List of all accelerators supported by SkyPilot.
+# Canonicalized names of all accelerators (except TPUs) supported by SkyPilot.
 # NOTE: Must include accelerators supported for local clusters.
+
+# 1. What if a name is in this list, but not in any catalog?
+# The name will be canonicalized, but the accelerator will not be supported.
+# Optimizer will print an error message.
+# 2. What if a name is not in this list, but in a catalog?
+# The name will be searched on the catalog with its case being ignored.
+# If a match is found, the name will be canonicalized to that in the catalog.
```

Comment:
In other words, is this list simply an optimization to short-circuit checking the catalog?  If so, should document here or point 3.

Reply:
Yes it is. Fixed the comment.